src/main/java/backend/Logic.java

```diff
@@ -127,9 +127,9 @@ public void refresh() {
         });
     }
 
-    public CompletableFuture<Boolean> getIssueMetadata(String repoId, List<Integer> issues) {
+    public CompletableFuture<Boolean> getIssueMetadata(String repoId, Map<Integer, String> issues) {
```

Comment:
In this case it doesn't, but being sure that nothing will change an object after it's copied is a good guarantee to have. Otherwise we'll need more ad-hoc ways to get around the inability to treat TurboIssues as values.